main.c

```diff
@@ -1,12 +1,27 @@
 // TODO(wendi): init external oscillator, then change this to 12MHz
 #include <stdint.h>
 #include "mcc_generated_files/mcc.h"
+
+#include "canlib/can_common.h"
+#include "canlib/pic18f26k83/pic18f26k83_can.h"
+#include "canlib/util/timing_util.h"
+#include "canlib/util/can_tx_buffer.h"
 #include "canlib/pic18f26k83/pic18f26k83_timer.h"
+
 #include "process_fill.h"
 #include "leds.h"
 
+#define MAX_LOOP_TIME_DIFF_ms 1000
+
 // variable that stores current fill level
 volatile uint8_t fill_level = 0;
+volatile uint8_t direction = 0;
+
+// Memory pool for CAN transmit buffer
+uint8_t tx_pool[100];
+
+// Don't need to receive messages?
```

Comment:
All boards should handle `LED_ON` and `LED_OFF` messages (which you're doing here) for debugging purposes, and because it looks cool when you hit a button on your laptop and LEDs on 10 separate PCBs all light up. But otherwise, no I don't think there's any messages you need to receive